app/assets/stylesheets/qsets.css.scss

```diff
@@ -1,3 +1,16 @@
+.no-qset-chosen {
+  color: gray;
+  font-style: italic;
+}
+
+.qset-chosen {
```

Comment:
As mentioned above, this is probably not needed at all, given it effectively just uses our app defaults.